tests/playwright/sanity/modules/nested-tabs/helper.ts

```diff
@@ -151,3 +151,77 @@ export async function isTabTitleVisible( context: Page | Frame, positionIndex: n
+		numberOfTitles = await nestedItemTitle.count(),
+		numberOfContents = await nestedItemContent.count();
+
+	// Act
+	await deleteItemButton.last().click();
+
+	await editor.getPreviewFrame().waitForSelector( `.elementor-element-${ widgetID }` );
+
+	// Assert
+	await expect.soft( nestedItemTitle ).toHaveCount( numberOfTitles - 1 );
+	await expect.soft( nestedItemContent ).toHaveCount( numberOfContents - 1 );
+}
+
+export async function addItemFromRepeater( editor: EditorPage, widgetID: string ) {
+	// Arrange
+	const addItemButton = editor.page.locator( '.elementor-repeater-add' ),
+		nestedItemTitle = editor.getPreviewFrame().locator( `.elementor-element-${ widgetID } .e-n-tab-title` ),
+		nestedItemContent = editor.getPreviewFrame().locator( `.elementor-element-${ widgetID } > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content > .e-con` ),
+		numberOfTitles = await nestedItemTitle.count(),
+		numberOfContents = await nestedItemContent.count();
+
+	// Act
+	await addItemButton.click();
+
+	await editor.getPreviewFrame().waitForSelector( `.elementor-element-${ widgetID }` );
+
+	// Assert
+	await expect.soft( nestedItemTitle ).toHaveCount( numberOfTitles + 1 );
+	await expect.soft( nestedItemContent ).toHaveCount( numberOfContents + 1 );
+}
+
+export async function cloneItemFromRepeater( editor: EditorPage, widget: string, position: number ) {
+	// Arrange
+	const nestedItemTitle = editor.getPreviewFrame().locator( `.elementor-element-${ widget } .e-n-tab-title` ),
+		nestedItemContent = editor.getPreviewFrame().locator( `.elementor-element-${ widget } .e-n-tabs-content > .e-con` ),
+		numberOfTitles = await nestedItemTitle.count(),
+		numberOfContents = await nestedItemContent.count(),
+		cloneItemButton = editor.page.locator( '.elementor-repeater-tool-duplicate' ).nth( position );
+
+	// Act
+	await cloneItemButton.click();
+
+	const currentTitle = nestedItemTitle.nth( position ),
+		currentTitleId = await currentTitle.getAttribute( 'id' ),
+		currentTitleIndex = await currentTitle.getAttribute( 'data-tab-index' ),
+		currentTitleText = await currentTitle.locator( '.e-n-tab-title-text' ).innerText(),
+		currentContainerId = await nestedItemContent.nth( position ).getAttribute( 'id' ),
+		currentContainerAriaLabeledBy = await nestedItemContent.nth( position ).getAttribute( 'aria-labelledby' );
+
+	const clonedItem = nestedItemTitle.nth( position + 1 ),
+		clonedTitleId = await clonedItem.getAttribute( 'id' ),
+		clonedTitleIndex = await clonedItem.getAttribute( 'data-tab-index' ),
+		clonedTitleText = await clonedItem.locator( '.e-n-tab-title-text' ).innerText(),
+		clonedContainerId = await nestedItemContent.nth( position + 1 ).getAttribute( 'id' ),
+		clonedContainerAriaLabeledBy = await nestedItemContent.nth( position + 1 ).getAttribute( 'aria-labelledby' );
+
+	// Assert
+	await expect( nestedItemTitle ).toHaveCount( numberOfTitles + 1 );
```

Comment:
```suggestion
	await expect( nestedItemTitle ).toHaveCount( nestedItemTitle.count()s + 1 );
```

Reply:
I don't think this is a good suggestion.
The test works the following way:
1. I count the number of Items in the tabs before cloning the elements.
2. I add an element
3. I check that in the nested element I have the initial number + 1

If I will implement your suggestion the count will give me the number of items after the clone action.